apps/etterlatte-saksbehandling-ui/client/src/components/person/sakOgBehandling/sakUtils.ts

```diff
@@ -32,3 +33,23 @@ const vedtakDatoStrengTilDate = (dato: string | undefined) => (dato ? new Date(d
 // å sjekke den for å erstatte med virkningstidspunkt
 export const ytelseOpphoersdato = (loependeVedtak: VedtakSammendrag) =>
   loependeVedtak.opphoerFraOgMed ? loependeVedtak.opphoerFraOgMed : loependeVedtak.virkningstidspunkt
+
+// TODO: flytte til en annen plass?
```

Comment:
Usikker på om det er rett plass å legge dette, men hjelpefunksjon for å hente saksid. Utfordringen er at vi må hente saksid fra ulike kontekster avhengig av hvor i Gjenny vi er

Reply:
ganskje legge det i ~shared?

Reply:
kanskje*